arkouda/numeric.py

```diff
@@ -606,17 +768,71 @@ def arctan2(num: Union[pdarray, numeric_scalars], denom: Union[pdarray, numeric_
                 },
             ),
         )
-    )
+        return create_pdarray(repMsg)
+    elif where is False:
+        if isinstance(num, pdarray) and isinstance(denom, pdarray):
+            ret = num / denom
+        if isinstance(num, pdarray) and not isinstance(denom, pdarray):
+            ret = num / denom
+        if isinstance(denom, pdarray) and not isinstance(num, pdarray):
+            ret = num / denom
+        return ret
```

Comment:
`mypy` has problems with just a return statement of `num / denom`. The previous logic ensures that we are in a case where either one or both are pdarrays, and it kept making the assumption that this division could lead to a non-array value.